daphne/src/hpke.rs

```diff
@@ -60,6 +63,104 @@ fn check_suite<T: HpkeCrypto>(
     }
 }
 
+/// Codepoint for KEM schemes compatible with HPKE.
```

Comment:
What's the motivation for moving the `HpkeConfig`? The reason it's in `messages` now is that this module contains all of the protocol messages.

Also, did the implementation change? It's kind of hard to tell as a result of the code movement (though I didn't look closely yet).

Reply:
there was no implementation change, the one thing I added was a `From<u16> for HpkeKemId|HpkeFdfId|HpkeAreaId`.

The reason I moved it out of there is because we have a dedicated hpke module, and hpke is used outside of the domain of messages (stored in kv, used for encryption and decryption, etc). So it felt odd of to have that be in messages. What makes sense to have in messages is the `Encode` and `Decode` implementations for these types, since this is a concern of serialization and not of cryptography. What I left in the messages module was the `HpkeConfigList` (which the diff poorly demonstrates).

Reply:
Ack. I'll defer to your preference here, but I think it would be best to remove this change and discuss it in another PR. As a general rule (at least in my opinion), code movement should be avoided unless clearly related to the PR.

In any case I carefully reviewed the changes and they look correct to me.